src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/adjourncase/AdjournCaseMidEventValidationHandler.java

```diff
@@ -2,7 +2,6 @@
 
 import static java.util.Objects.nonNull;
 import static uk.gov.hmcts.reform.sscs.ccd.domain.YesNo.isYes;
-import static uk.gov.hmcts.reform.sscs.util.DateTimeUtils.isDateInTheFuture;
 import static uk.gov.hmcts.reform.sscs.util.DateTimeUtils.isDateInThePast;
 
 import java.util.Set;
```

Comment:
I think we should rename this name this class so that it's clear how it's different from the other one